jira/src/getConfig.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Builds the Community Connector config.
+ * @return {Config} The Community Connector config.
+ * @see https://developers.google.com/apps-script/reference/data-studio/config
+ */
+function getConfig() {
+  var cc = DataStudioApp.createCommunityConnector();
+  var config = cc.getConfig();
+
+  config
+    .newInfo()
+    .setId('instructions')
+    .setText('Enter the following information to connect to Jira Cloud.');
+
+  config
+    .newTextInput()
+    .setId('host')
+    .setName('Host')
+    .setHelpText('Jira cloud host')
+    .setPlaceholder('examplehost.atlassian.net')
+    .setAllowOverride(true);
+
+  config
```

Comment:
This is fine for now, but you might want to look into the filter pushdown functionality (https://developers.google.com/datastudio/connector/filters) for a v2.

Reply:
Thank you! I will take a look at it.

Reply:
This has been addressed in the latest commit